src/app/containers/PageHandlers/withHashChangeHandler/isHashChangeOnSamePath.js

```diff
@@ -0,0 +1,10 @@
+export default (
```

Comment:
do you think this covers the case if some other props change at the same time as a hash change? I _think_ that's what the previous implementation checked for.

Reply:
so you would want to check that the **only** prop that has changed is the hash prop

Reply:
I've done a bit of testing on this and state/prop changes appear to update as expected. The social embeds re-render (causing that unstyled appearance at least for Twitter) in this case, which I think is what we want to happen as memoisation is not preventing those re-renders from happening.

Reply:
cool thanks for testing this